aya/src/obj/mod.rs

```diff
@@ -517,9 +549,19 @@ impl Object {
             BpfSectionKind::Btf => self.parse_btf(&section)?,
             BpfSectionKind::BtfExt => self.parse_btf_ext(&section)?,
             BpfSectionKind::Maps => {
-                let name = section.name.splitn(2, '/').last().unwrap();
-                self.maps
-                    .insert(name.to_string(), parse_map(&section, name)?);
+                let symbols = self
+                    .symbols_by_index
+                    .values()
+                    .cloned()
```

Comment:
Not that it makes much of a difference but cloned should probably be after filter